tools/aa/commands/WorkspaceCommand.java

```diff
@@ -0,0 +1,55 @@
+package com.google.startupos.tools.aa.commands;
+
+import com.google.startupos.common.FileUtils;
+import com.google.startupos.common.flags.Flag;
+import com.google.startupos.common.flags.FlagDesc;
+import com.google.startupos.common.flags.Flags;
+import java.io.IOException;
+import java.nio.file.Paths;
+import javax.inject.Inject;
+
+public class WorkspaceCommand implements AaCommand {
+  @FlagDesc(
+    name = "force",
+    description = "Force workspace switching creating workspace if non-existent"
```

Comment:
We wanted to support "-f" as well, right?
Our Flags library doesn't support it (and intentionally, since single-letter flags are more prone to be defined in multiple packages).
To support single-letter flags here, I suggest replacing "-f" with "--force" before passing on to `Flags.Parse()`.